packages/next-swc/crates/next-core/src/next_app/metadata/mod.rs

```diff
@@ -273,7 +273,8 @@ fn format_radix(mut x: u32, radix: u32) -> String {
         }
     }
 
-    result.into_iter().rev().collect()
+    result.reverse();
+    result[..6].iter().collect()
```

Comment:
Nit: I think you can
```suggestion
    result.into_iter().rev().take(6).collect()
```